etc/profile-m-z/strawberry.profile

```diff
@@ -13,14 +13,19 @@ noblacklist ${MUSIC}
 
 include disable-common.inc
 include disable-devel.inc
+include disable-exec.inc
 include disable-interpreters.inc
 include disable-passwdmgr.inc
 include disable-programs.inc
 include disable-xdg.inc
+include whitelist-usr-share-common.inc
 
-include whitelist-var-common.inc
```

Comment:
Why removing wvc?

BTW: you need to move wusc down (just add one blank line between disable-xdg and it).

Reply:
I thought I added it by mistake, but it worked without it, shall I add it again?

Reply:
Added it.

Reply:
With it, it is more secure because only files listed in it are allowed.